ci/dictionary.txt

```diff
@@ -0,0 +1,327 @@
+personal_ws-1.1 en 0 utf-8
```

Comment:
This file looks like it will become a challenge to maintain.  What is the plan to keep this up to date?

Reply:
Yeah I know that this is not ideal.
This is actually how rust-book manages spellchecking in their CI. 
A contributor would have to add missing words as lines to somewgere in the dictionary.txt . The spellchecker already uses aspell-en dictionary so most English vocabulary is already present and some technical lingo and variable names still need to be added.